src/views/Project/index.jsx

```diff
@@ -22,15 +41,57 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
```

Comment:
This probably needs to be `no-cache` right?

Reply:
@helfi92 but this one is for the github?

Reply:
Oh right. I guess we can leave this one `network-only`.